Styles/Components/badge.css

```diff
@@ -0,0 +1,82 @@
+.badge {
+  position: relative;
+}
+
+.badge .online {
+  background: var(--green-1);
+  border: 3px solid white;
+  border-radius: 50%;
+  position: absolute;
+  height: 1.25rem;
+  width: 1.25rem;
+}
+
+.badge .offline {
+  background: var(--red-1);
```

Comment:
You can use better or more descriptive name for colors. e.g Danger for Red, Success for Green.